src/_guides/language/analysis-options.md

```diff
@@ -0,0 +1,268 @@
+to perform static analysis.
+
+You can customize static analysis to look for a variety of potential
+problems, including errors and warnings specified in the
+[Dart language spec](/guides/language/spec).
+You can also configure the linter, one of the analyzer's plugins,
+to ensure that your code complies with the
+[Dart Style Guide](/guides/language/effective-dart/style)
+and other suggested guidelines
+in [Effective Dart](/guides/language/effective-dart).
+Dart tools such as the
+[Dart Dev Compiler (DDC)](https://github.com/dart-lang/dev_compiler),
+[`dartanalyzer`](https://github.com/dart-lang/sdk/tree/master/pkg/analyzer_cli#dartanalyzer),
+[`flutter analyze`](https://flutter.io/debugging/#the-dart-analyzer),
+and the [IntelliJ IDEs](https://www.dartlang.org/tools/jetbrains-plugin)
+use the analyzer package to evaluate your code.
+
+This document explains how to customize the behavior of the analyzer
+using an `.analysis_options` file. If you want to
+add static analysis to your tool, see the
+[analyzer package](https://pub.dartlang.org/packages/analyzer) docs and the
+[Analysis Server API Specification](https://htmlpreview.github.io/?https://github.com/dart-lang/sdk/blob/master/pkg/analysis_server/doc/api.html).
+
+
+## The analysis options file
+
+The `.analysis_options` (or `.analysis_options.yaml`) file,
+written in the YAML language, is typically placed in the same
+directory as the pubspec file at the top of the package.
+If no file is present, the analyzer walks up the directory
+tree, looking for one.
+
+Here's a sample YAML analysis_options file:
+
+{% prettify yaml %}
+analyzer:
+  strong-mode:
+    - no-implicit-downcasts: true
+    - no-implicit-dynamic: true
+  errors:
+    todo: ignore
+  exclude:
+    - flutter/**
+    - lib/api/*.dart
+
+linter:
+  rules:
+    - avoid_empty_else
+    - cancel_subscriptions
+    - close_sinks
+{% endprettify %}
+
+YAML is sensitive to whitespace&mdash;don't use tabs in a YAML file,
+and use 2 spaces to denote each level of indentation.
+
+<div class="alert alert-info" markdown="1">
+Note: You might come across a `language:` tag in an analysis options file.
+This tag is used for testing experimental features. You can ignore it.
+</div>
+
+## Specifying strong mode
+
+The Dart language spec supports dynamic typing, allowing you to
+write code that has no type annotations at all.
+Strong mode applies more restrictive rules to the type system and,
+as a result, finds more errors during static analysis and at runtime.
+Another benefit of strong mode is faster compilation.
+Some tools, such as DDC, require strong mode compliance.
+
+The simplest way to enable strong mode is to specify
+`strong-mode: true` in the analysis-options file:
+
+{% prettify yaml %}
+analyzer:
+  strong-mode: true
+{% endprettify %}
+
+This flag defaults to false. Instead of specifying `true`
+you can use the following flags to look for specific types
+of implicit casting, on top of the standard strong mode checks.
```

Comment:
Later you say that both flags default to true, which makes it sound like you're not *adding* to strong mode checks, but *removing* standard strong mode checks.

Or maybe the "This flag defaults to true" statements should be something like "This flag, when present, defaults to true"?